server-worker/src/test/scala/com/aliyun/emr/rss/service/deploy/integration/SkewJoinTest.scala

```diff
@@ -29,10 +29,11 @@ class SkewJoinTest extends SparkTestBase {
       .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
       .set("spark.sql.adaptive.autoBroadcastJoinThreshold", "-1")
       .set("spark.sql.autoBroadcastJoinThreshold", "-1")
+      .set("spark.sql.parquet.compression.codec", "gzip")
```

Comment:
spark parquet table uses snappy in default, not sure why snappy dependency is broken here, just switching to gzip as a workaround.